jaxlib/mosaic/gpu/target.cc

```diff
@@ -64,25 +64,42 @@ absl::StatusOr<std::pair<std::string, std::string>> GetSmAndPtxIsaVersion(
       }
     }
   }
+  return sm_arch_specific ? sm_arch_specific : sm_base;
+}
 
-  const std::string sm = sm_arch_specific ? sm_arch_specific : sm_base;
-
+absl::StatusOr<std::string> GetPtxIsaVersion(int major, int minor) {
```

Comment:
This is dead now, isn't it? Perhaps delete it?

Reply:
Sorry this PR is a bit raw, I'll refactor it a bit better + add more inline docs. This function does play a role here, it takes an input PTX ISA version specified as `major.minor` and returns `min(major.minor, <latest PTX ISA supported by LLVM>)`. This is to cover cases when ptxas is more recent and LLVM hasn't caught up to it yet.

Reply:
I refactored the function for clarity.